h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
+        model['is_moderator'] = True
```

Comment:
> My inclination would be to return "moderator": false in those cases, as it simplifies the work of any API client.

👍 - It makes no practical difference in JS because we'll just test the falsyness of `group.moderator` but for languages that don't have such lax treatment of missing properties it will be nicer and conceptually its better because it simplifies the type of this property from `boolean|undefined` to just `boolean`.